sql/mob_droplist.sql

```diff
@@ -27309,6 +27303,201 @@ INSERT INTO `mob_droplist` VALUES (3373,0,0,1000,922,@COMMON); -- Bat Wing (Comm
 INSERT INTO `mob_droplist` VALUES (3373,0,0,1000,930,@VRARE);  -- Vial Of Beastman Blood (Very Rare, 1%)
 INSERT INTO `mob_droplist` VALUES (3373,4,0,1000,922,0);       -- Bat Wing (Despoil)
 
+-- ZoneID:  73 - Puk
+INSERT INTO `mob_droplist` VALUES (3374,0,0,1000,5365,@ALWAYS);
+
+-- ZoneID:  73 - Ziz
+INSERT INTO `mob_droplist` VALUES (3375,0,0,1000,5375,@ALWAYS);
+
+-- ZoneID: 73 - Poroggo Gent
+INSERT INTO `mob_droplist` VALUES (3376,0,0,1000,5365,@ALWAYS);
+INSERT INTO `mob_droplist` VALUES (3376,0,0,1000,5365,@ALWAYS);
+INSERT INTO `mob_droplist` VALUES (3377,0,0,1000,5373,@ALWAYS);
+INSERT INTO `mob_droplist` VALUES (3377,0,0,1000,5373,@ALWAYS);
+INSERT INTO `mob_droplist` VALUES (3378,0,0,1000,5374,@ALWAYS);
+INSERT INTO `mob_droplist` VALUES (3378,0,0,1000,5374,@ALWAYS);
+
+-- ZoneID: 73 - Poroggo Madame
+INSERT INTO `mob_droplist` VALUES (3379,1,1,@COMMON,14962,200);
+INSERT INTO `mob_droplist` VALUES (3379,1,1,@COMMON,14978,200);
+INSERT INTO `mob_droplist` VALUES (3379,1,1,@COMMON,15638,200);
+INSERT INTO `mob_droplist` VALUES (3379,1,1,@COMMON,15720,200);
+INSERT INTO `mob_droplist` VALUES (3379,1,1,@COMMON,16089,200);
+
+-- ZoneID:  73 - Draco Lizard
+INSERT INTO `mob_droplist` VALUES (3380,0,0,1000,5370,0);
+
+-- ZoneID:  73 - Wyvern
+INSERT INTO `mob_droplist` VALUES (3381,0,0,1000,5366,@ALWAYS);
+INSERT INTO `mob_droplist` VALUES (3381,0,0,1000,5366,@ALWAYS);
+INSERT INTO `mob_droplist` VALUES (3381,0,0,1000,5367,@VCOMMON); -- Cumulus Cell (Vert Common, 24%)
+
+-- ZoneID:  73 - Mamool Ja Zenist
+INSERT INTO `mob_droplist` VALUES (3382,0,0,1000,5366,@VCOMMON);
+INSERT INTO `mob_droplist` VALUES (3382,0,0,1000,5370,@VCOMMON);
+
+-- ZoneID:  73 - Mamool Ja Spearman
+INSERT INTO `mob_droplist` VALUES (3383,0,0,1000,5369,@ALWAYS);
+INSERT INTO `mob_droplist` VALUES (3383,0,0,1000,5367,@VCOMMON);
+
+-- ZoneID:  73 - Mamool Ja Zenist
+INSERT INTO `mob_droplist` VALUES (3384,0,0,1000,5377,@ALWAYS);
+INSERT INTO `mob_droplist` VALUES (3384,0,0,1000,5377,@ALWAYS);
+INSERT INTO `mob_droplist` VALUES (3384,0,0,1000,5377,@ALWAYS);
+INSERT INTO `mob_droplist` VALUES (3384,0,0,1000,5383,@VCOMMON);
+INSERT INTO `mob_droplist` VALUES (3384,0,0,1000,5384,@VCOMMON);
+
+-- ZoneID:  73 - Mamool Ja Savant
+INSERT INTO `mob_droplist` VALUES (3385,0,0,1000,5370,0);
```

Comment:
No chance

Reply:
Its for a treasure listener to adjust treasure on conditions